src/routes/admin/[classId]/module/[moduleId]/states.svelte.ts

```
@@ -211,8 +251,10 @@ export class QuestionCurationState {
 			const url = new URL(window.location.href);
 			if (url.searchParams.has('edit')) {
 				url.searchParams.delete('edit');
-				const newPath = url.pathname + (url.search ? `?${url.searchParams.toString()}` : '');
-				await goto(newPath, { replaceState: true, noScroll: true });
+				await goto(url.pathname, {
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)